src/stores/process.ts

```diff
@@ -29,7 +32,7 @@ export interface CommandFactory<T = any, P extends object = DefaultPayload> {
  * Command that returns patch operations based on the command request
  */
 export interface Command<T = any, P extends object = DefaultPayload> {
-	(request: CommandRequest<T, P>): Promise<PatchOperation<T>[]> | PatchOperation<T>[];
+	(request: CommandRequest<T, P>): Promise<PatchOperation<T>[]> | PatchOperation<T>[] | void;
```

Comment:
I think this needs to be `void | Promise<void>` for async commands when using the `state` object.

Reply:
@maier49 Did you see this comment?

Reply:
@agubler Could have sworn I did that, sorry. It's done now!